core/eval.go

```diff
@@ -187,6 +187,19 @@ func evalSET(args []string) []byte {
 			if obj != nil {
 				return RESP_NIL
 			}
+		case "GET":
```

Comment:
Since we are adding GET, we cannot now blindly return from XX and NX.

Reply:
Actually, I'm not able to get it completely, could you please elaborate more on this

Reply:
Hey @soumya-codes , I was just now running the tests with the `XX` option with the `GET` one. The test is failing. I think do we need to change the complete `evalSET()` func? I'm thinking of an approach for this, will update soon about it here 👍